packages/client/src/components/Questions.tsx

```diff
@@ -212,25 +213,34 @@ const Questions: React.FC<
   let disableFutureQuestions = false;
 
   // Check which questions are causing the need for a permit
-  // @TODO: We can refactor this and move to checker.js
-  const permitsPerQuestion = [] as boolean[];
-  checker.isConclusive() &&
-    checker.permits.forEach((permit: Permit) => {
-      const conclusionDecision = permit.getDecisionById("dummy");
-      if (
-        conclusionDecision?.getOutput() === imtrOutcomes.NEED_PERMIT ||
-        conclusionDecision?.getOutput() === imtrOutcomes.NEED_CONTACT
-      ) {
+  // @TODO: Move this to `imtr-client`
+  const permitsPerQuestion = [] as string[];
+  checker.permits.forEach((permit: Permit) => {
+    const conclusionDecision = permit.getDecisionById("dummy");
+
+    if (conclusionDecision) {
+      let outcomeType = "";
+      if (conclusionDecision.getOutput() === imtrOutcomes.NEED_CONTACT) {
```

Comment:
Why not store `conclusionDecision.getOutput()` in a variable instead of calling it 3 times?